lib/cli/scripts/init-aps-env.ts

```diff
@@ -434,3 +435,24 @@ function sleep(delay) {
     const start = new Date().getTime();
     while (new Date().getTime() < start + delay) {  }
 }
+
+async function checkAcsEnv() {
+    try {
+        const repositoryEntry = await alfrescoJsApiRepo.oauth2Auth.callCustomApi(
```

Comment:
we have another script that jsut cehck ACS why don't just use that before instead of coupling and duplicate? the script is check-cs-env

Reply:
Do you mean I should use check-cs-env in the single pipeline script [initialize_services.sh](https://github.com/Alfresco/terraform-alfresco-pipeline/blob/cfc1f1190be3299bfc85e07bc94da3323f7882fb/scripts/travis/build/initialize_services.sh#L18) before initializing APS?

Reply:
yep